closure/repositories.bzl

```diff
@@ -680,11 +680,11 @@ def com_google_javascript_closure_library():
     http_archive(
         name = "com_google_javascript_closure_library",
         urls = [
-            "https://mirror.bazel.build/github.com/google/closure-library/archive/v20190415.tar.gz",
-            "https://github.com/google/closure-library/archive/v20190415.tar.gz",
+            "https://mirror.bazel.build/github.com/google/closure-library/archive/82b285261eedce6a52bf1d3b1667e16c6d10fe1e.tar.gz",
+            "https://github.com/google/closure-library/archive/82b285261eedce6a52bf1d3b1667e16c6d10fe1e.tar.gz",
         ],
-        sha256 = "b92f9c2b81adeb06b16e0a5d748baf115eeb58884fe864bec3fe8e23a7cf7d14",
-        strip_prefix = "closure-library-20190415",
+        sha256 = "61cbeac7c403b8d591f2edaa717faa90048787012de55442dad3c9d1f2dab7b2",
+        strip_prefix = "closure-library-82b285261eedce6a52bf1d3b1667e16c6d10fe1e",
```

Comment:
I'm with @davido, can you update this PR for `v20190819`.

Someone will also need to upload the archive to `mirror.bazel.build`.